src/EditorFeatures/CSharpTest/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionFixAllTests.cs

```diff
@@ -0,0 +1,203 @@
+                {
+                    case 10:
+                        return 10;
+                    case 20:
+                        return 20;
+                    case 30:
+                        return 30;
+                    case var _:
+                        return 0;
+                }
+            case 3:
+                switch (j)
+                {
+                    case 10:
+                        return 10;
+                    case 20:
+                        return 20;
+                    case 30:
+                        return 30;
+                    case var v:
+                        return 0;
+                }
+        }
+    }
+}",
+@"class Program
+{    
+    int M(int i, int j)
+    {
+        var r = i switch
+        {
+            1 => 1,
+            2 => 2,
+            3 => 3,
+            _ => 4,
+        };
+        int x, y;
+        switch (i)
+        {
+            case 1:
+                x = 1;
+                y = 1;
+                break;
+            case 2:
+                x = 1;
+                y = 1;
+                break;
+            case 3:
+                x = 1;
+                y = 1;
+                break;
+            default:
+                x = 1;
+                y = 1;
+                break;
+        }
+        switch (i)
+        {
+            default:
+                throw null;
+            case 1:
+                return j switch
+                {
+                    10 => 10,
+                    20 => 20,
+                    30 => 30,
+                    _ => 0,
+                };
+            case 2:
+                return j switch
+                {
+                    10 => 10,
+                    20 => 20,
+                    30 => 30,
+                    var _ => 0,
+                };
+            case 3:
+                return j switch
+                {
+                    10 => 10,
+                    20 => 20,
+                    30 => 30,
+                    var v => 0,
+                };
+        }
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        public async Task TestNested_02()
+        {
+            await TestInRegularAndScriptAsync(
+@"class Program
+{
+    System.Action<int> M(int i, int j)
+    {
+        {|FixAllInDocument:switch|} (i)
+        {
+            default:
+                return () =>
+                {
+                    switch (j)
+                    {
+                        default:
+                            return 3;
+                    }
+                };
+        }
+    }
+}",
+@"class Program
+{
+    System.Action<int> M(int i, int j)
+    {
+        return i switch
+        {
+            _ => () =>
+                 {
+                     switch (j)
+                     {
+                         default:
+                             return 3;
+                     }
+                 }
+            ,
```

Comment:
I have no idea where this trivia is coming from.

Reply:
Fortunately, there are debuggers :D